lib/dom/utils.ts

```diff
@@ -15,3 +15,7 @@ export async function waitForDomSettle() {
 }
 
 window.waitForDomSettle = waitForDomSettle;
+
+export function calculateViewportHeight() {
+  return Math.ceil(window.innerHeight * 0.75);
```

Comment:
ESLint might still throw an error here, we can cross that bridge when we get to it